metadata-ingestion/src/datahub/ingestion/source/fivetran/config.py

```diff
@@ -60,29 +63,42 @@ class Constant:
 }
 
 
-class DestinationConfig(BaseSnowflakeConfig):
+class SnowflakeDestinationConfig(BaseSnowflakeConfig):
     database: str = Field(description="The fivetran connector log database.")
     log_schema: str = Field(description="The fivetran connector log schema.")
 
 
+class BigQueryDestinationConfig(BigQueryConnectionConfig):
+    dataset: str = Field(description="The fivetran connector log dataset.")
+
+
 class FivetranLogConfig(ConfigModel):
     destination_platform: str = pydantic.Field(
         default="snowflake",
         description="The destination platform where fivetran connector log tables are dumped.",
     )
-    destination_config: Optional[DestinationConfig] = pydantic.Field(
+    snowflake_destination_config: Optional[SnowflakeDestinationConfig] = pydantic.Field(
```

Comment:
this change is backwards incompatible - use the pydantic rename helpers to make it compatible

Reply:
Can you please provide example for this

Reply:
Look for usages of `pydantic_renamed_field` in our codebase

Reply:
also please make sure to add a test for this config handling